plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -39,7 +71,9 @@ class HealthChecksFormSection extends Component {
             <FormGroup
               className="column-3"
               showError={Boolean(errors.gracePeriodSeconds)}>
-              <FieldLabel>Grace Period (s)</FieldLabel>
+              <FieldLabel helpText={gracePeriodHelpText}>
```

Comment:
Arh! I missed that part. However, I feel this is going against our pattern of composing using smaller components. I would like us to do this as suggested above